### DISCUSSION TITLE
Não consigo fazer push na master - pt-Br

### DISCUSSION THREAD

**Discussion by connectiva** [community/community#65706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Olá. Voltei a usar o git a pouco tempo. Não consigo fazer um comite na master do meu repositorio.

git push origin master

Grato

**Comment by dnatividade** [comment]
Olá @connectiva 

Tente:

```
git push origin main
```

**Reply by connectiva** [reply]
Deu certo. Obrigado.
